public/index.html

```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
```

Comment:
If we hover over todo tasks , cursor should be pointer .
![image](https://user-images.githubusercontent.com/89918408/215976416-4ff9d20e-9345-4d02-9392-832eed81dc05.png)